docs/conf.py

```diff
@@ -44,6 +45,10 @@
     'sphinxcontrib.images',
 ]
 
+# 'releases' (changelog) settings
+releases_issue_uri = "https://github.com/raiden-network/raiden/issues/%s"
+releases_release_uri = "https://github.com/raiden-network/raiden/tree/%s"
```

Comment:
Shouldn't this be `https://github.com/raiden-network/raiden/releases/tag/%s`?

Reply:
I also thought so. Perhaps, but just took it from the [example conf.py](https://github.com/fabric/fabric/blob/4afd33e971f1c6831cc33fd3228013f7484fbe35/docs/conf.py#L31) linked from their [usage guide](http://releases.readthedocs.io/en/latest/usage.html).

Pethaps should change it ... not sure how it's used, would have to dig a bit deeper.

Reply:
As far as I understand this is used to construct the URL from the `:release:` directives

Reply:
yes that seems to be the case. I will adjust the value.